scripts/protocgen.sh

```diff
@@ -35,4 +35,4 @@ rm -rf github.com
 
 go mod tidy
 
-./scripts/protocgen2.sh
\ No newline at end of file
+# ./scripts/protocgen2.sh
```

Comment:
commented this out cause it doesn't work as intended. We should fix in a follow up pr.

Reply:
I think it's fixed now? `make proto-gen` works well for me